BUILD.bazel

```diff
@@ -65,6 +88,24 @@ go_image(
     x_defs = {"main.gitCommit": "{STABLE_GIT_COMMIT}"},
 )
 
+# Initialize the default /data storage directory since the bazel-remote
+# user does not have sufficient permissions at the root (/) directory.
+# Since an empty directory cannot be packaged by the pkg_tar rule, create
+# the "EMPTY" file.
+genrule(
+    name = "empty",
+    srcs = [],
+    outs = ["EMPTY"],
+    cmd = "touch $@",
+)
+
+pkg_tar(
+    name = "data_tar",
+    srcs = [":empty"],
+    package_dir = "/data",
```

Comment:
I'm getting a permission denied issue because I had my volume mounted at /cache instead of /data -- does this change affect the dir-path?

Reply:
> I'm getting a permission denied issue because I had my volume mounted at /cache instead of /data -- does this change affect the dir-path?

When you start the docker container you need to map an existing cache dir in the host system that the docker user has read/write/execute permissions on to `/data` inside the container, in your case something like `-v /cache:/data`.